extensions/azurecore/src/account-provider/auths/azureAuth.ts

```diff
@@ -364,10 +364,19 @@ export abstract class AzureAuth implements vscode.Disposable {
 				};
 				return this.handleInteractionRequiredMsal(tenant, resource);
 			} else if (e.name === 'ClientAuthError') {
-				Logger.error(e.message);
+				Logger.error('[ClientAuthError] Failed to silently acquire token');
+				return {
+					name: e.name,
+					errorCode: e.errorCode,
+					errorMessage: e.errorMessage || e.message
+				}
+			} else {
+				return {
```

Comment:
Did you purposely not want to log anything in this case?

Given that these are both returning the same thing consider combining them into one and then doing an if check inside if you need.